eng/Versions.props

```diff
@@ -7,6 +7,7 @@
   </PropertyGroup>
   <!-- Production Dependencies -->
   <PropertyGroup>
+    <HumanizerCorePackageVersion>2.2.0</HumanizerCorePackageVersion>
     <PrivateSourceBuildReferencePackagesPackageVersion>1.0.0-beta.20562.2</PrivateSourceBuildReferencePackagesPackageVersion>
```

Comment:
Merge conflict with adjacent line.

(I think it would be reasonable to put this in a new propertygroup just to force Git to ignore this in the future. Adjacency doesn't seem to me to be a very useful heuristic for meaningful conflicts in this file. 😛)